docs/paper_examples.md

```diff
@@ -0,0 +1,70 @@
+### Paper Examples
+
+Contained below are the toy examples from the *Usage Examples* section of the [`foyer` paper](https://arxiv.org/pdf/1812.06779.pdf).
+The source code selections are listed below on this page, there are [Jupyter Notebooks](https://github.com/mosdef-hub/foyer/tree/master/docs/examples) where you can try these examples yourself.
+Note that these examples are meant to showcase the abilities of `foyer` through rather simplistic examples.
+If the user would like to examine more in-depth and complex examples, please refer to the [tutorial repository](https://github.com/mosdef-hub/mosdef_tutorials).
+
+
+Below is *Listing 6* from the paper, a python script to fill a $$2x2x2 nm$$ box with 100 ethane molecules.
+The system is then atomtyped using the OPLS-AA forcefield.
+There are two approaches to the same problem detailed below in this listing, the first approach uses the `forcefield_files` function argument from [`mBuild`](https://github.com/mosdef-hub/mbuild) to atomptype the system (using foyer under the hood).
+While the second approach creates a `foyer` `Forcefield` object, which then calls its `apply` function, operating on the `mBuild` `Compound` to return the properly atomtyped structure.
```

Comment:
Again with defining that's it a `ParmEd` structure, as distinct from one of ours